### PR TITLE
Feature/logrus

### PR DESCRIPTION
**Pull request by ransford**
Switch to [logrus](https://github.com/sirupsen/logrus/) for leveled logging and revisit a few decisions around logging and output.

### CODE REVIEW

**Comment by ransford** [comment]
Hmm, now noticing a "I can't maintain this project anymore" message in logrus's readme. Consider [zap](https://github.com/uber-go/zap) instead?

File: hl7_decode.go [hunk 1]
```diff
@@ -123,18 +125,18 @@ func (decoder *HL7Decoder) DecodePayload(app *gopacket.ApplicationLayer) (string
 	// non-ASCII data in the string:
 	//
 	// Print a raw Payload with escaped non-ASCII printing characters:
-	// logger.Printf("%+q\n", string(app.Payload()))
+	// log.Printf("%+q\n", string(app.Payload()))
```
Comment (ukrutt) [inline hl7_decode.go]: Probably `log.Debugf` here?  (I can't figure out how to "suggest a change" via Github.com.)

**Comment by ukrutt** [comment]
Ooops, just saw the comment on maintenance/"zap"

**Comment by ransford** [comment]
Closing this in favor of a similar approach that I'll fold into `feature/refactor-main`.

**Pull request closed without merging**